src/api/app/models/token/release.rb

```diff
@@ -7,16 +7,20 @@ def self.token_name
   end
 
   def call(_options)
+    # release from remote is not supported
+    raise ActiveRecord::RecordNotFound if @token.package_from_association_or_params.nil?
```

Comment:
This class will never be instatiated if this is nil. It'll crash in the Token::ReleasePolicy. See [here](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/policies/token/release_policy.rb#L10).

Reply:
we would not run into this, the release operation needs to have a package object (unlike rebuild).

but it needs to operation on the multibuild container only.